mungegithub/publisher/deployment.yaml

```diff
@@ -10,6 +10,25 @@ spec:
         app: publisher
         target-repo: @@
     spec:
+      initContainers:
```

Comment:
After the change, every time I deploy the publisher on a new cluster, `mungegithub/mungers/publish_scripts/initialize_repos.sh` will have to run, which takes a very long time. Does that bother you?

Reply:
The script is only slow if the volume is new. You will usually keep the volume alive and only redeploy the deployment.